src/neo_stocktwits_data_extractor.py

```diff
@@ -70,7 +76,7 @@ def main():
             print('scrolling... ',no_of_pagedowns, ' to go')
 
         time_between_refreshes = 30
-        refresh_attempts = 120*70
+        refresh_attempts = optionalArgs.time*2
```

Comment:
What is the logic of multiplying time by 2?

Reply:
time isn't actually minutes - there's 30 seconds between refresh attempts, and "time" refresh attempts, so we multiply that by two to approximate "time" actually being the number of minutes the script runs for

Reply:
Got it. Could you add a comment to the code that it is an approximation. It will act as a reminder later